components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -209,12 +216,29 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 
 	_clearSelected() {
 		this._selectedEntries = [];
+		this._hideClearFiltersButton = true;
```

Comment:
For consistency, I would set this to `this._hideClearFiltersButton = this._selectedEntries.length < CLEAR_FILTERS_THRESHOLD;`

If a change to design is ever made where we always want to show the button, the obvious change would be to change CLEAR_FILTERS_THRESHOLD to something like -1, but this would still cause the button to disappear

Reply:
Actually it looks like you just put the condition straight into the control, so you can probably just remove this line

Reply:
Good point. However I'm actually going to remove `_hideClearFiltersButton` all together, Jeff pointed out that it's not really needed anymore

Reply:
I got rid of `_hideClearFiltersButton `, in the process I also removed the `_refreshFilters` function